src/Http/Http/src/Features/FormFeature.cs

```diff
@@ -326,7 +326,10 @@ private static bool HasMultipartFormContentType([NotNullWhen(true)] MediaTypeHea
 
     private bool ResolveHasInvalidAntiforgeryValidationFeature()
     {
-        if (_request is null) return false;
+        if (_request is null)
```

Comment:
We may want to try updating the nullability annotations on the `_request` field as well here.

Reply:
Yes, that should probably be done too. It would be cleaner. But then there are a few more null checks missing in this type, I guess...

Reply:
Yep -- it expands the scope of the PR a bit, but I figure this is the right time to do it while we have a motivating scenario.

If you don't have capacity to make the change, I'm happy to take a stab at it.

Reply:
I can give it a go. Happy to contribute.